packages/xo-web/src/xo-app/xoa/update/index.js

```diff
@@ -122,9 +123,11 @@ const Updates = decorate([
           effects.update(),
         ])
       },
-      initialize() {
+      initialize: async effects => {
```

Comment:
Better use a state full method like the previous implementation.

Reply:
What's a stateful method?

Reply:
May bad, i want to say regular function.

Reply:
As opposed to arrow function or async function?

Reply:
My comment is related to the translation of the regular function to the arrow function. 

I do this comment because we said to prefer using regular functions and changing the context using `this.state` instead of returning the state.